CMakeLists.txt

```diff
@@ -14,7 +15,14 @@ include_directories(include)
 add_library(msd_channel INTERFACE)
 target_include_directories(msd_channel INTERFACE include)
 
-enable_testing()
-add_subdirectory(tests)
+option(cpp_channel_build_tests "Build all of cpp_channel's own tests." OFF)
```

Comment:
With this option OFF by default, tests are not running in the GitHub workflow because they are not built anymore. Please set this option to ON in [cmake.yml](https://github.com/andreiavrammsd/cpp-channel/blob/master/.github/workflows/cmake.yml#L59).

Reply:
Should I also turn the examples build option on? it can help to detect errors in example code.